### DISCUSSION TITLE
NestJS: How to create Base CRUD Service with Prisma

### DISCUSSION THREAD

**Discussion by ngoc199** [prisma/prisma#10025] [Q&A] [answered]
CRUD is a very basic function, so I don't want to duplicate it multiple times.

How could I achieve this with Prisma?

**Comment by austincrim** [comment, marked as answer]
Hey @ngoc199 👋!

Reusability and extensibility are active areas of research as we look to improve Prisma in the future. I don't have a single answer or example for your question, but [this issue](https://github.com/prisma/prisma/issues/5273) has a lot of discussion and potential solutions, specifically for NestJS. I would encourage you to read through the thread and add your +1/use case in the thread.

If you have more specific questions, I would be happy to help point you in the right direction!

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
